### PR TITLE
Enhance GpsDataCapturer UI

### PR DESCRIPTION
**Pull request by lynnliu655**
# Description
UI enhancement. 

## Type of change
* Split GPS Data text view into Lat/Lon/Speed separate parts with individual live data 
* Split GPS Status text view and its live data
* Split Satellite text view and its live data
* Change mobile from relative layout to constraint layout and made it works for both portrait and landscape mode
  *   image
  *. image

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: GpsDataCapturer/mobile/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,7 +4,14 @@
...
+    <string name="gps_data" translatable="false">GPS Data</string>
     <string name="waiting_gps_data" translatable="false">Waiting for GPS Data...</string>
-    <string name="gps_status_not_available">GPS Status: Not Available</string>
-    <string name="satellites_not_available">Satellites: Not Available</string>
+    <string name="gps_status_not_available" translatable="false">Not Available</string>
+    <string name="satellites_not_available" translatable="false">Not Available</string>
+    <string name="satellites" translatable="false">Satellites:  </string>
+    <string name="gps_status" translatable="false">GPS Status:  </string>
+    <string name="lat" translatable="false">LAT:  </string>
+    <string name="loc_data" translatable="false">00.000000</string>
```
Comment (AdsuleA) [inline GpsDataCapturer/mobile/src/main/res/values/strings.xml]: should this be called as default location data?

File: GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java [hunk 2]
```diff
@@ -15,26 +16,40 @@
...
+    public String getLongitude(){
+        return this.longitude;
+    }
+
+    /**
+     * Get the speed of the GpsData
+     * @return speed
      */
     @NonNull
-    public String getGpsDataString() {
```
Comment (AdsuleA) [inline GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java:29]: lets keep this method for future debugging

File: GpsDataCapturer/wear/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -8,7 +8,13 @@
...
-    <string name="waiting_gps_data" translatable="false">Waiting for GPS Data...</string>
-    <string name="gps_status_not_available">GPS Status: Not Available</string>
-    <string name="satellites_not_available">Satellites: Not Available</string>
+    <string name="waiting_gps_data" translatable="false">Waiting for GPS Data...\n</string>
+    <string name="gps_status_not_available">Not Available</string>
+    <string name="satellites_not_available">Not Available</string>
+    <string name="satellites">Satellites:  </string>
+    <string name="gps_status">GPS Status:  </string>
+    <string name="lat">LAT:  </string>
+    <string name="loc_data">00.000000</string>
```
Comment (AdsuleA) [inline GpsDataCapturer/wear/src/main/res/values/strings.xml]: can we rename it to default_location_data?

File: GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java (hunk 2 above)
Comment (lynnliu655) [reply GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java:29]: How about change this method to ` @Override toString() ` method and make GpsData to immutable value object too?

File: GpsDataCapturer/wear/src/main/res/values/strings.xml (hunk 3 above)
Comment (lynnliu655) [reply GpsDataCapturer/wear/src/main/res/values/strings.xml]: Done.

File: GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java (hunk 2 above)
Comment (AdsuleA) [reply GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java:29]: Sounds good.

Comment (lynnliu655) [reply GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java:29]: Done.

File: GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java [hunk 4]
```diff
@@ -1,40 +1,102 @@
...
+   * Get the string of GpsData
+   *
+   * @return a string of GpsData
+   */
+  @NonNull
+  @Override
+  public String toString() {
+    StringBuilder gpsDataBuilder = new StringBuilder();
+    gpsDataBuilder
+        .append("GPS DATA \n")
```
Comment (AdsuleA) [inline GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java]: let's avoid newlines as a single log would span across multiple lines.

File: GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java [hunk 5]
```diff
@@ -1,40 +1,102 @@
...
+        .append("Lat: ")
```
Comment (AdsuleA) [inline GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/model/GpsData.java]:
```suggestion
        .append(" Lat: ")
```

**Review by AdsuleA — APPROVED** [review]

**Pull request merged**